.github/workflows/validate_python.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Setup python3
       uses: actions/setup-python@v2
       with:
-        python-version: '3.x'
+        python-version: '3.7.x'
```

Comment:
Note: I had a surprisingly hard time getting my local build to use python `3.7` even with pyenv set up (I believe) correctly. I think this is something about the `FindPython` module in cmake (with pyenv? on windows?). Anyway, I confirmed this worked.

Reply:
Yeah, I'm not sure how CMake searches for python by default. It's probably not aware of the pyenv directories. Maybe try `Python_ROOT_DIR` as a command line arg for CMake on your local machine and give it the pyenv path or similar?

Reply:
Yeah, that would probably work. But `pyenv local/global` set up your PATH so `python` is pointed to the pyenv dir, right? So it seemed more like `cmake` is "trying to hard".